### PR TITLE
Add cogification for one dataset

### PR DESCRIPTION
**Pull request by gadomski**
**Related Issue(s):**
None

**Description:**
Adds cogification for one CDR dataset, ocean-heat-content. This should be _mostly_ reusable for other datasets but I'm sure there will be edge cases.

**PR checklist:**

- [x] Code is formatted (run `scripts/format`).
- [x] Code lints properly (run `scripts/lint`).
- [x] Tests pass (run `scripts/test`).
- [x] Documentation has been updated to reflect changes, if applicable.

### CODE REVIEW

File: src/stactools/noaa_cdr/commands.py [hunk 1]
```diff
@@ -63,6 +67,7 @@ def create_item_command(source: str, destination: str) -> None:
     def create_download_command(name: str, destination: str) -> None:
```
Comment (pjhartzell) [inline src/stactools/noaa_cdr/commands.py:67]: I like this download command. Will likely pilfer this for future datasets. 🙂

**Review by pjhartzell — APPROVED** [review]
Looks good. Tests pass. 👍
